corehq/apps/case_search/models.py

```diff
@@ -6,21 +6,55 @@
 from django.db import models
 from django.forms import model_to_dict
 
+import attr
 from jsonfield.fields import JSONField
 
+from corehq.apps.case_search.exceptions import CaseSearchUserError
 from corehq.util.quickcache import quickcache
 
 CLAIM_CASE_TYPE = 'commcare-case-claim'
 FUZZY_PROPERTIES = "fuzzy_properties"
 CASE_SEARCH_BLACKLISTED_OWNER_ID_KEY = 'commcare_blacklisted_owner_ids'
 CASE_SEARCH_XPATH_QUERY_KEY = '_xpath_query'
-CASE_SEARCH_REGISTRY_ID_KEY = 'commcare_registry'
-CASE_SEARCH_EXPAND_ID_PROPERTY_KEY = 'commcare_expand_id_property'
+CONFIG_KEY_PREFIX = "x_commcare_"
```

Comment:
Late thoughts on naming:
* Do the config keys really need a different prefix? We use just "commcare" for `commcare_blacklisted_owner_ids`, `commcare_search_score`, and in custom fields: https://github.com/dimagi/commcare-hq/blob/179a5c166f76f9f8830c8a5072d4a025aa0a0a49/corehq/apps/custom_data_fields/models.py#L24, and also `commcare_search_score` 
* Do you think `_xpath_query` should also use this prefix?

Reply:
Quick thoughts:
- `commcare_blacklisted_owner_ids` and `_xpath_query` are special search terms
- `commcare_search_score` is something we return in the results set like `commcare_project` so they are quite different

We certainly don't need the prefix, I just thought it made it clearer that these are not search terms.